lib/openhab/core/configuration.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module OpenHAB
+  module Core
+    java_import org.openhab.core.config.core.Configuration
+
+    # Adds methods to core openHAB {org.openhab.core.config.core.Configuration} to make it more natural in Ruby
+    class Configuration
+      # inspect result is just the string representation
+      alias_method :inspect, :to_s
+
+      # Return the configuration value for the given key
+      # @!method [](key)
+      # @param [String] key
+      # @return [Object]
+      alias_method :[], :get
+
+      # Check for content equality
+      alias_method :==, :equals
```

Comment:
I thought JRuby already does this alias automatically

Reply:
I had to reimplement it anyway to do the key stringification